gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProvider.java

```diff
@@ -186,42 +186,11 @@ private ManagedChannel createSingleChannel() throws IOException {
 
     // TODO(hzyi): Change to ManagedChannelBuilder directly when
```

Comment:
Could you delete this TODO as well? It's resolved by this change as well.

Reply:
for sure